### DISCUSSION TITLE
Jahresertrag - Kalender umstellen

### DISCUSSION THREAD

**Discussion by Werni61** [tbnobody/OpenDTU#1636] [Q&A]
Hi,
ich benutze die ätere Firmware dd3c12d mit 0.1.25 da es nach der Aktualisierung zu Problemen gekommen ist, hier läuft aber alles bestens. Nur eine Sache, bei mir wird über die Webseite immer noch das Kalenderjahr 2023 angezeigt, von selbst aktualisiert sich da nichts und ich finde auch nirgend eine Einstellung dazu. Was kann ich tun? Vielen Dank schon mal.

**Comment by stefan123t** [comment]
@Werni61 "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."
Der o.a. commit zu GoogleCharts ist von @RaBa64 und nicht Teil des OpenDTU master branches.

Andererseits ist Deine Version von August 2023 und wir haben beinahe laufend neue Versionen, vielleicht mal die aktuelle Version von OpenDTU-Database GoogleCharts branch einspielen und damit das Problem nochmal verifizieren.

Folglich kann Dir zu seinem Fork / PR wahrscheinlich auch am Besten RaBa64 selbst Auskunft geben.
Eventuell machst Du ein Issue in seinem Repository https://github.com/RaBa64/OpenDTU-Database auf ?

**Reply by Werni61** [reply]
OK, vielen Dank, schaue ich mir mal an.
